daemon/cluster/executor/container/container.go

```diff
@@ -207,7 +206,7 @@ func getMountMask(m *api.Mount) string {
 func (c *containerConfig) hostConfig() *enginecontainer.HostConfig {
 	return &enginecontainer.HostConfig{
 		Resources: c.resources(),
-		Binds:     c.bindMounts(),
+		Binds:     append(c.bindMounts(), c.volumes()...),
```

Comment:
Double check this works with an empty source...

When I tried to implement this, volumes that were just the dst/target were required to be in `Volumes`.